documentation/source/troubleshooting.rst

```diff
@@ -15,14 +15,51 @@ Increasing Verbosity
 If things fail in the VPI/VHPI/FLI area, check your simulator's documentation to see if it has options to
 increase its verbosity about what may be wrong. You can then set these options on the :command:`make` command line
 as :make:var:`COMPILE_ARGS`, :make:var:`SIM_ARGS` or :make:var:`EXTRA_ARGS` (see :doc:`building` for details).
+Moreover the scheduler activity can be enhanced by setting the variable :make:var:`COCOTB_SCHEDULER_DEBUG` to
+one.
```

Comment:
"activity can be enhanced" is a weird way to phrase this. Perhaps
```rst
If things fail from within python, or coroutines aren't being called when you expect, the :make:var:`COCOTB_SCHEDULER_DEBUG` variable can be used to (greatly) increase the verbosity of the scheduler.
```

Reply:
Yeah I forgot to double check. Sure it does not make sense as I wrote it.
I will commit the suggestion.